src/main/resources/application.yml

```diff
@@ -72,3 +88,39 @@ app:
 spring:
   profiles: auth
 
+---
+###############################################################################
+# Profile - "dev"
+###############################################################################
+spring.profiles: dev
```

Comment:
could we use docker-compose to handle those adjustments? Because when we deploy to QA/DEV cluster, we don't use profile like `dev` or `qa` since that's not flexible. We set all the values we need through helm which helm is very good at, instead of set some magic profile. I think just `test` profile would be enough, what do you think?